CMakeLists.txt

```diff
@@ -0,0 +1,19 @@
+cmake_minimum_required(VERSION 3.6)
+
+if (WIN32 AND DEFINED ENV{VCPKG_ROOT})
```

Comment:
This should not be here. The entire point of Vcpkg is that the build scripts can be agnostic of using Vcpkg or not. Vcpkg is not Windows only, so there is no reason to check for targeting Windows or not.